src/flamenco/vm/fd_vm_interp_core.c

```diff
@@ -170,6 +170,8 @@
   goto interp_exec; /* Silly but to avoid unused label warning in some configurations */
 interp_exec:
 
+  FD_VM_INTERP_INSTR_EXEC;
```

Comment:
`FD_VM_INTERP_INSTR_EXEC` performs a jump, which means the tracing stuff below this can never get executed